clouddriver-kubernetes/src/test/java/com/netflix/spinnaker/clouddriver/kubernetes/op/job/KubernetesRunJobOperationTest.java

```diff
@@ -74,50 +72,28 @@ void setTask() {
   @Test
   void deploysJobWithName() {
     KubernetesRunJobOperationDescription runJobDescription = baseJobDescription("job.yml");
-    OperationResult result = operate(runJobDescription, false);
+    OperationResult result = operate(runJobDescription);
 
     assertThat(result.getManifestNamesByNamespace()).containsOnlyKeys(NAMESPACE);
     assertThat(result.getManifestNamesByNamespace().get(NAMESPACE))
         .containsExactlyInAnyOrder(DEPLOYED_JOB);
   }
 
-  @Test
-  void deploysJobWithNameAndAddsSuffix() {
-    KubernetesRunJobOperationDescription runJobDescription = baseJobDescription("job.yml");
-    OperationResult result = operate(runJobDescription, true);
-
-    assertThat(result.getManifestNamesByNamespace()).containsOnlyKeys(NAMESPACE);
-    assertThat(result.getManifestNamesByNamespace().get(NAMESPACE)).hasSize(1);
-    String job =
-        Iterators.getOnlyElement(result.getManifestNamesByNamespace().get(NAMESPACE).iterator());
-    assertThat(job).startsWith(DEPLOYED_JOB);
-    String suffix = job.substring(DEPLOYED_JOB.length());
-    assertThat(suffix).startsWith("-");
-    // We're asserting that at least 4 characters are added after the hyphen; rather than
-    // overspecify the test by looking up exactly how many are added, we're just making sure that a
-    // reasonable number to guarantee randomness are added.
-    assertThat(suffix.length()).isGreaterThanOrEqualTo(5);
-  }
-
-  @ParameterizedTest
-  @ValueSource(booleans = {true, false})
-  void deploysJobWithGenerateName(boolean appendSuffix) {
+  void deploysJobWithGenerateName() {
```

Comment:
I don't think this test is going to run without the `@Test` annotation

Reply:
Right you are: https://github.com/spinnaker/clouddriver/pull/5310/commits/11177f72c5350cd10616473e3d2f2db6782f2b1c